### PR TITLE
ignore generic binary from github tree

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
I start to noded this binary generated during the build process and maybe this can be excluded!

Signed-off-by: Vincenzo Palazzo <vincenzopalazzodev@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -67,4 +67,7 @@ doc/lightning*.[1578]
 
 # Rust targets
 target
-Cargo.lock
\ No newline at end of file
+Cargo.lock
+
+# other binary
+fp16
```
Comment (laanwj) [inline .gitignore:73]: `devtools/.gitignore` would be a better place for this

Comment (vincenzopalazzo) [reply .gitignore]: Good catch thanks!

**Comment by ZmnSCPxj** [comment]
ACK 0b2f7854640d8045e80168c69301fa32ee4d65b6

**Comment by ZmnSCPxj** [comment]
Merge in, changelog not needed.

**Pull request merged**